File: Arrays/Easy/2. 2nd Largest Element in the Array/main.js
let nums = [1, 2, 4, 7, 7, 5];

function secondLargest(nums) {
  let f = 0,
    s = 0;
  for (let num of nums) {
    if (num > f) {
      s = f;
      f = num;
    }
    if (num > s && num < f) {
      s = num;
    }
  }
  return s;
}
console.log(secondLargest(nums));

File: Arrays/Easy/2. 2nd Largest Element in the Array/main.go
package main

import "fmt"

func main() {
	nums := []int{1, 2, 4, 7, 7, 5}
	var first, second int
	for _, num := range nums {
		if num > first {
			second = first
			first = num
		}
		if num > second && num < first {
			second = num
		}
	}
	fmt.Println(second)
}
